src/login/systemd-user.in

```diff
@@ -5,7 +5,7 @@
 {% if ENABLE_HOMED %}
 -account sufficient pam_systemd_home.so
 {% endif %}
-account sufficient pam_unix.so
+account sufficient pam_unix.so no_pass_expiry
```

Comment:
this definitely looks right, given we don't really care about passwords at all for this service, its a background service without any password prompting afte rall